tools/ci-scripts/linux/image-cleanup/collect_images.py

```diff
@@ -2,15 +2,15 @@
 
 """Main entry point for garbage image cleanup."""
 
-from datetime import timedelta
+import argparse
 import logging
 import subprocess
 import sys
+
+from datetime import timedelta
 from typing import Iterable
 
-from collector.acr import (
-    delete_image_by_manifest,
-    get_image_manifests)
+from collector.acr import (delete_image_by_manifest, get_image_manifests)
```

Comment:
~ You don't need these parens - I think the purpose they had been serving was to continue the line.